api/migration.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (C) 2015-2019, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+from typing import Dict
+import json
+import os
+import re
+import yaml
+
+from wazuh import common
+
+from api import validator
+
+
+def get_old_config() -> Dict:
+    """
+    Gets variables from old API
+    :param old_config_path: Path of old API configuration file
+    :return: Dictionary with the variables from 'config.js' file
+    """
+    old_config_path = os.path.join(common.ossec_path, '~api/configuration/config.js')
+    old_config = {}
+    regex = re.compile(r'^\s*config.(\w+)\s*=\s*\"?([\w\/.]*|true|false)\"?;?$')
+    try:
+        # import old config file!
+        with open(old_config_path) as input_file:
+            for line in input_file:
+                match = regex.match(line)
+                if match:
+                    var_name = match.group(1)
+                    var_value = match.group(2)
+                    if check_old_config({var_name: var_value}):
+                        # add element to old_config only if it is right
+                        old_config[var_name] = parse_to_yaml_value(var_value)
+    except IOError:
+        raise
```

Comment:
Why catch an exception to re-raise it?

Reply:
This exceptions need to be redefined, I know :). I want to know the new format of error messages first.